src/main/java/frc/robot/subsystems/HangSubsystem.java

```diff
@@ -33,15 +34,30 @@ public HangSubsystem() {
     liftMotor.setSmartCurrentLimit(Constants.CURRENT_LIMIT);
     winchMotor.setSmartCurrentLimit(Constants.CURRENT_LIMIT);
 
+    liftMotor.setIdleMode(IdleMode.kBrake);
+    liftMotor.setIdleMode(IdleMode.kCoast);
+
     liftEncoder = liftMotor.getEncoder();
     liftEncoder.setPosition(0);
```

Comment:
I know this isn't part of the PR, but we should really just:

```suggestion
  resetEncoder();
```

since we have that defined